pkg/eventpersisterdwh/storage/v2/sql/count_experiment.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+    (
+        SELECT
+            count(*)
+        FROM
+            experiment
+        WHERE
+            status = 1
+    ) + (
+        SELECT
+            count(*)
+        FROM
+            experiment
+        WHERE
+            status = 2
+          AND stop_at > UNIX_TIMESTAMP(
```

Comment:
@Ubisoft-potato, can you fix the indent in these two lines?

![Screenshot 2024-01-29 at 4 05 38 PM](https://github.com/bucketeer-io/bucketeer/assets/2486691/6cff08c3-fe53-486d-a59f-2799a57d4d42)

Reply:
fixed.